packages/xo-web/src/common/intl/messages.js

```diff
@@ -2204,6 +2204,9 @@ const messages = {
   srAllDisconnected: 'Disconnected',
   srSomeConnected: 'Partially connected',
   srAllConnected: 'Connected',
+  maintenanceSrModalBody:
+    'In order to put this SR in maintenance mode, VM{n, plural, one {} other {s}} bellow will be shut down. Are you sure you want to continue?',
+  maintenanceMode: 'Maintenance mode',
```

Comment:
Is this only for the label next to the SR name? I think "Maintenance" is enough.

Reply:
Also used for the [modal title](https://github.com/vatesfr/xen-orchestra/pull/6308/files#diff-b89389078c64b467b5da403c33e0dc38a1a5d8c4ca2241ae78159616fbbde235R2121).